<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExperimentDialog</class>
 <widget class="QWidget" name="ExperimentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Experiment</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="6">
    <widget class="QPushButton" name="saveAsButton">
     <property name="text">
      <string>save as...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QSpinBox" name="repetitionSpinBox"/>
   </item>
   <item row="3" column="6">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="8" column="6">
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>run</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Repetitions</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="loadButton">
     <property name="text">
      <string>load</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" colspan="4">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
